wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/SendCommitBundleActionHandler.swift

```diff
@@ -1,192 +0,0 @@
-                return
-            }
-
-            let updateEvents = eventsData.compactMap { eventData in
-                ZMUpdateEvent(
-                    uuid: nil,
-                    payload: eventData,
-                    transient: false,
-                    decrypted: false,
-                    source: .download
-                )
-            }
-
-            action.succeed(with: updateEvents)
-
-        case (400, "mls-welcome-mismatch"):
-            action.fail(with: .mlsWelcomeMismatch(message: message))
-
-        case (400, "mls-group-conversation-mismatch"):
-            action.fail(with: .mlsGroupConversationMismatch(message: message))
-
-        case (400, "mls-client-sender-user-mismatch"):
-            action.fail(with: .mlsClientSenderUserMismatch(message: message))
-
-        case (400, "mls-self-removal-not-allowed"):
-            action.fail(with: .mlsSelfRemovalNotAllowed(message: message))
-
-        case (400, "mls-commit-missing-references"):
-            action.fail(with: .mlsCommitMissingReferences(message: message))
-
-        case (400, "mls-protocol-error"):
-            action.fail(with: .mlsProtocolError(message: message))
-
-        case (400, _):
-            action.fail(with: .invalidRequestBody(message: message))
-
-        case (403, "missing-legalhold-consent"):
-            action.fail(with: .missingLegalHoldConsent(message: message))
-
-        case (403, "mls-missing-sender-client"):
-            action.fail(with: .mlsMissingSenderClient(message: message))
-
-        case (403, "legalhold-not-enabled"):
-            action.fail(with: .legalHoldNotEnabled(message: message))
-
-        case (403, "access-denied"):
-            action.fail(with: .accessDenied(message: message))
-
-        case (404, "mls-proposal-not-found"):
-            action.fail(with: .mlsProposalNotFound(message: message))
-
-        case (404, "mls-key-package-ref-not-found"):
-            action.fail(with: .mlsKeyPackageRefNotFound(message: message))
-
-        case (404, "no-conversation"):
-            action.fail(with: .noConversation(message: message))
-
-        case (404, "no-conversation-member"):
-            action.fail(with: .noConversationMember(message: message))
-
-        case (409, "mls-stale-message"):
-            action.fail(with: .mlsStaleMessage)
-
-        case (409, "mls-client-mismatch"):
-            action.fail(with: .mlsClientMismatch)
-
-        case (409, _):
-            guard
-                let payload = FederationErrorResponse(response),
-                let nonFederatingDomains = payload.nonFederatingBackends
-            else {
-                let errorInfo = response.errorInfo
-                return action.fail(with: .unknown(
-                    status: response.httpStatus,
-                    label: errorInfo.label,
-                    message: errorInfo.message
-                ))
-            }
-            action.fail(with: .nonFederatingDomains(Set(nonFederatingDomains)))
```

Comment:
I guess we need to port this logic to MLSAPI implementation, see https://staging-nginz-https.zinfra.io/v9/api/swagger-ui/#/default/mls-commit-bundle